apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/modules/components/Claimable.tsx

```diff
@@ -57,20 +64,10 @@ export type ClaimConditionValue = {
   auxData: string;
 };
 
```

Comment:
Need to show a message when no claim condition is set for a token id

Reply:
addressed